go-controller/pkg/cluster/node.go

```diff
@@ -76,17 +76,25 @@ func (cluster *OvnClusterController) StartClusterNode(name string) error {
 	var cidr string
 	var connected bool
 
+	var wg sync.WaitGroup
+	type waitResult struct {
+		Error error
+	}
+
 	for _, clusterSubnet := range config.Default.ClusterSubnets {
 		clusterSubnets = append(clusterSubnets, clusterSubnet.CIDR.String())
 	}
 
+	node, err = cluster.Kube.GetNode(name)
```

Comment:
@mccv1r0 I think we actually want to move this back into the PollImmediate() just below, because we do need to wait for the node object itself to be created in the Kube API when ovnkube node starts up. This won't be the case in an OpenShift cluster (since we run as a daemonset and that requires the Node to be created so the kubelet can find the pod to start), but could be the case when run as a static pod or standalone.

Since you need the `node` variable just declare it before the PollImmediate and assign to it inside the PollImmediate and it'll be set to the right thing when the PollImmediate is done.